OpenQA/Isotovideo/Dewebsockify.pm

```diff
@@ -0,0 +1,125 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+
+package OpenQA::Isotovideo::Dewebsockify;
+
+use Mojo::Base -strict, -signatures;
+use Mojo::IOLoop::Server;
+use Mojo::IOLoop::Stream;
+use Mojo::Log;
+use Mojo::UserAgent;
+
+sub main ($args) {
+    die "Arguments must be a hash reference\n" unless defined $args && ref($args) eq 'HASH';
```

Comment:
The "\n" would mean one wouldn't see where this die would be triggered. is that intended?

Reply:
hmm, i had added that line for debugging and thought it'd maybe make sense to keep anyways, but you're right the \n does not make sense here. I'll remove it.